specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1,11 +1,13 @@
 openapi: 3.0.0
 servers:
-  - description: "Brk-bevragen Kadastraal Onroerende Zaak - PRODUCTIE"
+  - description: "Haal Centraal BRK-Bevragen API - PRODUCTIE"
     url: https://api.brk.kadaster.nl/esd/bevragen/v1
-  - description: "Brk-bevragen Kadastraal Onroerende Zaak - TEST"
+  - description: "Haal Centraal BRK-Bevragen API - TEST (via API-key & mTLS)"
     url: https://api.brk.kadaster.nl/esd-eto/bevragen/v1
+  - description: "Haal Centraal BRK-Bevragen API - TEST (via API-key)"
+    url: https://api.brk.kadaster.nl/esd-eto-apikey/bevragen/v1
 info:
-  title: Kadastraal onroerende zaken
+  title: Kadaster - Haal Centraal BRK-Bevragen API
```

Comment:
Het is even de vraag of de term "Haal Centraal" opgenomen moet worden in de naam van de API en van de servers. 
Haal Centraal is een programma met een kop en een staart. Doelstelling is dat we nog heel lang van deze API's genieten als het programma is afgelopen.

Reply:
@JohanBoer: Goed punt. Heb de term 'Haal Centraal' uit de API en servers description gehaald.